File: backend/src/main/java/hiddencore/ddasum/backend/web/dto/guardian/activephoto/ActivityGalleryListResponse.java
package hiddencore.ddasum.backend.web.dto.guardian.activephoto;

import java.util.List;

import lombok.Builder;

@Builder
public record ActivityGalleryListResponse(
        int slotCapacity,
        int filledCount,
        List<GalleryModalDto> slots,
        GalleryModalDto hero,
        List<GalleryModalDto> today,
        List<GalleryModalDto> week) {}

File: android/app/src/main/java/com/ddasum/app/data/remote/dto/ActivityGalleryListResponse.kt
package com.ddasum.app.data.remote.dto

/**
 * 백엔드 ActivityGalleryListResponse를 그대로 반영한 DTO
 * (GET /api/guardian/me/patients/{patientId}/activity-gallery).
 */
data class ActivityGalleryListResponse(
    val slotCapacity: Int,
    val filledCount: Int,
    val slots: List<GalleryCardDto>,
    val hero: GalleryCardDto?,
    val today: List<GalleryCardDto>,
    val week: List<GalleryCardDto>
)
